styles/SimpleNav.module.scss

```diff
@@ -0,0 +1,92 @@
+@import "styles/mixins.scss";
+@import "styles/variables/colors.module.scss";
+
+
+.navContainer {
+  display:flex;
+  justify-content: center;
+  flex-wrap:wrap-reverse;
+  align-items: center;
+  width:100%;
+  padding: 0rem 0rem;
+  @include for-tablet-up {
+    padding: 3rem 8rem;
+    justify-content: space-between;
+  }
+}
+
+.logoContainer {
+  width: 100%;
+  cursor: pointer;
+  padding:2rem;
+  @include for-tablet-up {
+    width: 16rem;
+    height: 82px;
```

Comment:
No biggie, but any reason to not use `rem` units here?

Reply:
its the exact value of the height, so it doesn't jump around when navigating to different pages with the same navigation header